src/main/java/build/buf/protovalidate/internal/evaluator/EvaluatorBuilder.java

```diff
@@ -226,16 +233,40 @@ private FieldEvaluator buildField(
         FieldDescriptor fieldDescriptor, FieldConstraints fieldConstraints)
         throws CompilationException {
       ValueEvaluator valueEvaluatorEval = new ValueEvaluator();
+      boolean ignoreDefault =
```

Comment:
Updates here are mostly equivalent to https://github.com/bufbuild/protovalidate-go/compare/v0.5.2...2268789d9be1f269f0001d64f5dd1bdd60278964#diff-70b43cf25409483d9785a2bea15d9d185795803e255330e6562e7dca5112663e (there are some differences in the Java vs. Go generated code).